README.md

```diff
@@ -4,13 +4,16 @@ Example of embedding Bare in an Android application using <https://github.com/ho
 
 ## Building
 
-To keep the build process fast and efficient, the project relies on a Bare Kit prebuild being available in the [`app/libs/`](app/libs) directory. Prior to building the project, you must therefore either clone and compile Bare Kit from source, or download the latest prebuild from GitHub. The latter is easily accomplished using the [GitHub CLI](https://cli.github.com):
+To keep the build process fast and efficient, the project relies on a Bare Kit prebuild being available in the [`app/libs/`](app/libs) directory. 
+Prior to building the project, you must therefore either clone and compile Bare Kit from source, or download the latest prebuild from GitHub.
+The latter is easily accomplished using the [GitHub CLI](https://cli.github.com):
```

Comment:
As a middleground I propose the bl for each end of sentence. WDYT?

Reply:
That makes for unnecessarily many visual jumps though. Either a tool hard wraps automatically at a fixed line length or people soft wrap at their preferred line length.